server/services/star.js

```diff
@@ -460,7 +503,7 @@ module.exports = class StarService extends EventEmitter {
             throw new Error('The star cannot be reignited, it is not dead.');
         }
 
-        star.naturalResources = naturalResources || 1;
+        star.naturalResources = naturalResources || {economy: 1, industry:1, science:1};
```

Comment:
Why do we do this?

Reply:
I mean, when the value for re ignition isn't given it needs a base value anyway. That is the {economy: 1, industry: 1, science: 1}... Practically it is always given though, so this should not change much